registries/_format/byte.md

```diff
@@ -12,10 +14,104 @@ layout: default
 
 Base type: `{{ page.base_type }}`.
 
-The `{{page.slug}}` format represents any sequece of octets encoded as a base64 string. This format entry is to ensure future versions of OpenAPI maintain compatibility with [OpenAPI 3.0.x](https://spec.openapis.org/oas/v3.0.0).
+The `{{page.slug}}` format represents any sequence of octets encoded as a base64 string as defined in [RFC4648](https://www.rfc-editor.org/rfc/rfc4648#section-4). This format entry is to ensure future versions of OpenAPI maintain compatibility with [OpenAPI 3.0.x](https://spec.openapis.org/oas/v3.0.0).
+
+### Upgrading from < 3.1 to >= 3.1
+
+#### Entire response body
+
+```yaml
+#OAS 3.0
+paths:
+  '/picture':
+    get:
+      responses:
+        '200':
+          content:
+            image/png:
+              schema:
+                type: string
+                format: byte
+
+#OAS 3.1
+paths:
+  '/picture':
+    get:
+      responses:
+        '200':
+          content:
+            image/png:
+              type: string
+              contentEncoding: base64
+```
+
+#### Encoded property as part of a structured response
+
+```yaml
+#OAS 3.0
+paths:
+  '/picture':
+    get:
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                properties:
+                  width:
+                    schema:
+                      type: string
+                  height:
+                    schema:
+                      type: string
+                  image:
+                    type: string
+                    format: byte
+
+#OAS 3.1
+paths:
+  '/picture':
+    get:
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                properties:
+                  width:
+                    schema:
+                      type: string
+                  height:
+                    schema:
+                      type: string
+                  image:
+                    type: string
+                    contentMediaType: image/png
+                    contentEncoding: base64
+```
+
+#### Query or Path parameters
+
+Using byte or base64 for path or query parameters is not recommended since this encoding makes use of characters that are not URL safe.
```

Comment:
Since `base64url` is being newly added for 3.0, it might be worth noting the consequences and necessary workarounds:

> Data encoded using `byte` or `base64` will require URL percent-encoding before it can be used in a path or query parameter.  Use `base64url` instead to avoid this problem.

Reply:
updated